### PR TITLE
Fix as per https://github.com/bitcoinjs/bip32-wallet/issues/7

### PR DESCRIPTION
**Pull request by devluxg**
See title, sorry for the delay.  I had to fix some repo stuff before I could patch.
This PR adds the ability to create a new wallet from a hex string.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -55,7 +55,9 @@ Wallet.fromSeedBuffer = function (seed, network) {
 
   return new Wallet(external, internal)
 }
-
+Wallet.prototype.fromSeedHex = function (hex, network) {
+  return Wallet.fromSeedBuffer(new Buffer(hex, 'hex'), network);
```
Comment (dcousens) [inline index.js]: no semicolon please :)

File: index.js [hunk 2]
```diff
@@ -55,7 +55,9 @@ Wallet.fromSeedBuffer = function (seed, network) {
 
   return new Wallet(external, internal)
 }
```
Comment (dcousens) [inline index.js]: new line here

**Comment by dcousens** [comment]
If you could also add a line to `test/index.js` which tests this method, that'd be great.
Otherwise I'll add it...

**Comment by devluxg** [comment]
I hate to admit this, but in all honesty I'm not sure what's required to add a test for this and don't have the bandwidth to add it at this time.  Sorry about that.

**Comment by devluxg** [comment]
Also off topic, but what's the deal with semicolons?  I realize that they are technically optional, but they're part of the language and not using them can lead to unintuitive behavior.  What was the logic for that decision?

**Comment by dcousens** [comment]
@devluxg we use https://github.com/feross/standard,  its required to make the tests pass.

Closing this PR and making a new one that has the required changes...

**Pull request closed without merging**